bin/crypto_conditions_test_gen/conditions.py

```diff
@@ -0,0 +1,1217 @@
+#!/usr/bin/env python
+
```

Comment:
could we have a one-line description of what this script is for and if/when it should be executed?